specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/firewallPolicy.json

```diff
@@ -34,6 +34,215 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/firewallPolicies/{firewallPolicyName}/draft/default/deploy": {
+      "post": {
+        "tags": [
+          "FirewallPolicies",
+          "FirewallPoliciesDrafts"
+        ],
+        "operationId": "FirewallPoliciesDrafts_Deploy",
+        "description": "Deploy the specified Firewall Policy draft to Azure Firewall Policy.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "firewallPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Firewall Policy.",
+            "pattern": "^[a-zA-Z0-9]*$",
```

Comment:
Is the regex pattern correct? Policy name can contain other characters (e.g. - or _)

Reply:
I need to fix the regex. It apparently cannot start with a - or _. I'll double check the backend